package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshExporter.cs

```diff
@@ -238,6 +238,11 @@ public static class MeshExporter {
             sample.points[i] = UnityTypeConverter.ChangeBasis(sample.points[i]);
             if (sample.normals != null && sample.normals.Length == sample.points.Length) {
               sample.normals[i] = UnityTypeConverter.ChangeBasis(sample.normals[i]);
+            }
+			if (sample.tangents != null && sample.tangents.Length == sample.points.Length) {
```

Comment:
Looks like a tabs vs. spaces issue here.

Reply:
Ah, that's what I get for switching text editors for a moment. I have pushed a fix.